src/interpreter/utils.ts

```diff
@@ -153,20 +154,11 @@ export const setParamArgs = (
   return nameValueMappings;
 };
 
-const typeOf = (v: Value): string => {
-  if (v === null) {
-    return 'null';
-  } else if (Array.isArray(v)) {
-    return 'array';
-  } else {
-    return typeof v;
-  }
-};
-
+const typeOf = (v: Value): string => typeof v;
```

Comment:
These are not equivalent. For example, `typeof null` returns `object` while I believe calling `typeof` on an array returns `object` as well. Is it intended?

Reply:
I removed it because I don't think we are supposed to encounter null or array at the moment? Array maybe later but not null right? When we encounter then I'll add it back.

Reply:
Ok, that makes sense.